superset/annotation_layers/api.py

```diff
@@ -86,7 +88,9 @@ class AnnotationLayerRestApi(BaseSupersetModelRestApi):
         "descr",
         "created_by.first_name",
         "changed_by.first_name",
+        "changed_on",
         "changed_on_delta_humanized",
+        "created_on",
```

Comment:
Why do we need `changed_on`? it's better to use `changed_on_delta_humanized`, also if we need/want `created_on` we should add a `created_on_delta_humanized`